label_sync/labels.yaml

```diff
@@ -593,6 +593,12 @@ default:
       target: both
       prowPlugin: label
       addedBy: anyone
+    - color: d2b48c
+      description: Categorizes an issue or PR as relevant to ug-big-data.
+      name: ug/big-data
+      target: both
+      prowPlugin: label
+      addedBy: anyone
```

Comment:
Can you add a previously for `sig/big-data` to this?

Reply:
Done. Also added `deleteAfter` for `sig/big-data`.

Reply:
Actually, just remove the whole label section for sig/big-data. We don't want to delete the label. It'll be auto migrated.